ResearchKit/Common/ORKOrderedTask+ORKPredefinedActiveTask.h

```diff
@@ -635,6 +635,11 @@ NS_ASSUME_NONNULL_BEGIN
                                         trailType:(ORKTrailMakingTypeIdentifier)trailType
                                           options:(ORKPredefinedTaskOption)options;
 
+NS_ASSUME_NONNULL_END
```

Comment:
Why move the `NS_ASSUME_NONNULL_END` ? You can also just define your methods as nullable and it will be fine without moving the wrapper macro.

Reply:
My bad, I didn't start off by explicitly specifying the return type as nullable. I will revert this.